src/common/i18n/translations/en.ts

```diff
@@ -106,7 +106,8 @@ export const en: Translations = {
   },
   notEligibleScreen: {
     notEligible: "Not eligible",
-    logAppeal: "Please log an appeal request.",
+    logAppeal:
+      "Not eligible to collect any item. Contact your in-charge to find out about the appeal guideline.",
```

Comment:
Small nit, guideline or guidelines?

Reply:
changed to guidelines

Reply:
@shermainelim might be good to run through Eida before changing this in the future, since it's something she introduced.